AndroidHomeApplication/app/src/main/res/layout/item_movies_list.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="170dp"
```

Comment:
android:layout_width="match_parent"
android:layout_height="match_parent"

and use
tools: layout_width, tools:layout_height for preview